grpc-native/src/main/java/org/ballerinalang/net/grpc/nativeimpl/client/FunctionUtils.java

```diff
@@ -412,7 +400,7 @@ public static Object externStreamingExecute(BObject clientEndpoint, BString meth
                 DefaultStreamObserver responseObserver = new DefaultStreamObserver(Runtime.getCurrentRuntime(),
                         callbackService, semaphore);
                 StreamObserver requestSender;
-                DataContext context = new DataContext(Scheduler.getStrand(), null);
+                DataContext context = new DataContext(env, null);
```

Comment:
If we can add a new constructor that accepts only the `Environment` param in the datacontext, then we could easily get rid of the null. May be an improvement which could be done in the future

Reply:
In this scenario, we don't need to make async. We only need to pass the environment. So I make the future null.